packages/rum-core/src/domain/rumEventsCollection/error/trackReportError.ts

```diff
@@ -0,0 +1,21 @@
+import type { Observable, RawError } from '@datadog/browser-core'
+import { clocksNow, ErrorHandling, ErrorSource, initReportObservable, ReportType } from '@datadog/browser-core'
+
+export function trackReportError(errorObservable: Observable<RawError>) {
+  const subscription = initReportObservable([ReportType.csp_violation, ReportType.intervention]).subscribe(
+    (reportError) =>
+      errorObservable.notify({
+        startClocks: clocksNow(),
+        message: reportError.message,
+        stack: reportError.stack,
+        source: ErrorSource.REPORT,
```

Comment:
The rum events format will need to be updated

Reply:
Should not it be done before this PR?

Reply:
Done